packages/icons/src/logout.svg

```diff
@@ -0,0 +1,9 @@
+<!-- 
+  Based on Material design icons by Google - https://github.com/google/material-design-icons
+  which are licensed under the Apache License 2.0. Icon 'add_circle_outline (rounded)'.
```

Comment:
Maybe it is not based on "add_circle_outline" icon?

Reply:
ops... too much copy-paste... Thought that I changed that, but seems like I missed to add that change.,..